q100viz/grid.py

```diff
@@ -74,8 +74,8 @@ def read_scanner_data(self, message):
                 for x, cell in enumerate(row):
                     cell.id, cell.rot = array[y * self.y_size + x]
 
-                    # object with ID 3 selects cells
-                    cell.selected = cell.id == 3
+                    # any non-white object selects cells
+                    cell.selected = cell.id != 0
```

Comment:
was ist mit -1?

Reply:
ja.. eher so:
```suggestion
                    cell.selected = cell.id > 0
```

Reply:
ah, jetzt weiß ich wieder.. Da der LEGO-Scanner sich derzeit nicht all zu präzise skalieren lässt, habe ich -1 akzeptiert, um jedes nicht-komplett-weiße Feld als "selected" anzuerkennen.
Beispiel:
![Screenshot from 2021-08-24 15-41-38](https://user-images.githubusercontent.com/23742320/130627216-724b1e04-792d-4ff5-ab44-5cea60512d09.png)
→ wird nicht in den tags aufgeführt, soll aber erstmal erkannt werden.